ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPool.java

```diff
@@ -375,17 +389,6 @@ private Optional<BlockHeader> getChainHeadBlockHeader() {
     return blockchain.getBlockHeader(blockchain.getChainHeadHash());
   }
 
-  private Wei minTransactionGasPrice(final Transaction transaction) {
-    return getChainHeadBlockHeader()
-        .map(
-            chainHeadBlockHeader ->
-                protocolSchedule
-                    .getByBlockNumber(chainHeadBlockHeader.getNumber())
-                    .getFeeMarket()
-                    .minTransactionPriceInNextBlock(transaction, chainHeadBlockHeader::getBaseFee))
```

Comment:
this minTransactionPriceInNextBlock method doesn't get called anywhere else - do we not need it?

Reply:
good catch removed